docs/examples/events.py

```diff
@@ -0,0 +1,23 @@
+from nameko.events import EventDispatcher, event_handler
+from nameko.rpc import rpc
+
+class ServiceA(object):
+    """ Event dispatching service.
+    """
+    name = "service_a"
```

Comment:
I figured it was better to help distinguish between the arbitrary choices. a/b work together, as do x/y